.gitignore

```diff
@@ -1 +1,5 @@
+/debug
 /target
+Cargo.lock
+
+.vscode
```

Comment:
Ensure there is a newline at end of file.

Reply:
Why doesn't VS Code do this for me automatically m98 I'm not used to doing this manually like a pleb